src/style.css

```diff
@@ -1,3 +1,82 @@
+:root {
+  --light-blue: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family as a fallback. For example, you could have 'Roboto, Arial, sans-serif' instead of 'Roboto, Arial, serif'.